### PR TITLE
Disable MSAA for pixel_perfect example

### PR DESCRIPTION
**Pull request by lewiszlw**
# Objective

- Fixes https://github.com/bevyengine/bevy/issues/9170

### CODE REVIEW

**Comment by mockersf** [comment]
Disabling msaa doesn't really feel like fixing the problem but more hiding it. This doesn't happens for everyone, but only for some users on linux I think

If that's the way we want to go, a comment is needed to explain why this is needed, and on which platforms

**Comment by lewiszlw** [comment]
Added comment.

**Review by Selene-Amanita — APPROVED** [review]

File: examples/2d/pixel_perfect.rs [hunk 1]
```diff
@@ -4,6 +4,9 @@ use bevy::prelude::*;
 
 fn main() {
     App::new()
+        // MSAA is known to cause issues for some users on Linux.
```
Comment (mockersf) [inline examples/2d/pixel_perfect.rs:7]:
```suggestion
        // MSAA is known to cause issues for Linux users with an Intel integrated GPU.
```

File: examples/2d/pixel_perfect.rs [hunk 2]
```diff
@@ -4,6 +4,9 @@ use bevy::prelude::*;
...
+        // Disable MSAA util the issues are fixed. https://github.com/bevyengine/bevy/issues/3593
```
Comment (mockersf) [inline examples/2d/pixel_perfect.rs:8]:
```suggestion
        // Disable MSAA until the issues are fixed. https://github.com/bevyengine/bevy/issues/3593
```

**Comment by mockersf** [comment]
I won't approve this PR because I don't think it's a good thing to do, but if we want to do it this PR is good

**Comment by cart** [comment]
Yeah I agree with @mockersf. I don't think this is the right call. This is a bug to be fixed. Not something to be papered over in examples.

**Pull request closed without merging**